include/ext/optional.hpp

```diff
@@ -17,14 +17,13 @@
 #ifndef EXT_OPTIONAL_HPP
 #define EXT_OPTIONAL_HPP
 
-#include <boost/optional.hpp>
+#include <optional>
 
 namespace ext {
 
-// TODO: the following 'using' directive can be changed to
-//   using std::optional;
-// once MySQL source code switches to c++17
-using boost::optional;
+// this using declaration left only for compatibility reasons
```

Comment:
Same comment about this header. If we are concerned about 3rd-party code using this header then I guess it is better to explicitly mention them. If probability of existing of such code is low, then IMO it is better simply to remove this header completely.

Reply:
Likewise, comment updated

Reply:
OK.